queries/java/highlights.scm

```diff
@@ -105,8 +105,11 @@
 ((field_access
   object: (identifier) @type)
   (#lua-match? @type "^[A-Z]"))
-((scoped_identifier
-  scope: (identifier) @type)
+(scoped_identifier
+  scope: (identifier) @type
+  (#lua-match? @type "^[A-Z]"))
+(scoped_identifier
+  name: (identifier) @type
```

Comment:
why not just remove the field and capture (identifier) once

Reply:
agreed.